### PR TITLE
Ssl

### PR DESCRIPTION
**Pull request by reflectivedevelopment**

### CODE REVIEW

**Comment by ianco** [comment]
Looks good to me, there are conflicts though ...

**Comment by reflectivedevelopment** [comment]
I've updated it to avoid conflicts.

In the process, I've added websocket support, an alternative tunnel to ngrok, and added defaults to docker-compose.yml.

File: README.md [hunk 1]
```diff
@@ -84,9 +84,22 @@ docker-compose up
 You can also run the docker container with ngrok for testing purposes by running.
 
 ```
-docker-compose -f docker-compose-ngrok.yml up
+docker-compose -f docker-compose.ngrok.yml up
```
Comment (ianco) [inline README.md:87]: I think these instructions are outdated, no?  ngrok is the default when you run `./manage start` and to run with an nginx tunnel you run `./manage start --tunnel`?

**Review by ianco — APPROVED** [review]
Very slick!  I think the documentation needs updated (there are some outdated instructions in the README) but they can be cleaned up in a separate PR (it's my fault that the old instructions are still there :-( )

**Comment by swcurran** [comment]
@dbluhm -- this PR got lost.  Can you please take a look and approve/merge if you are happy? 

@ianco @reflectivedevelopment -- see the comment above from Ian about ngrok.  Are we good with this?

**Review by dbluhm — APPROVED** [review]
To unblock this PR, I vote for doc updates to be done in a follow up PR

**Pull request merged**